src/peakrdl_uvm/__peakrdl__.py

```diff
@@ -11,7 +12,13 @@
 
 class Exporter(ExporterSubcommandPlugin):
     short_desc = "Generate a UVM register model"
+    
+    cfg_schema = {
+        "user_template_dir": schema.DirectoryPath(),
+        "user_template_context": schema.FixedMapping(),
```

Comment:
The `FixedMapping` constructor requires a schema to be passed in. Also I dont think you want to use that one here. Probably using the `UserMapping` would be more appropriate here, since keys can be anything.

Reply:
Thank you very much for your reminder!